docs/docs/topics/load-balancing.md

```diff
@@ -21,7 +21,9 @@ policy:
 ```
 
 ::: tip
-    In presence of multiple upstreams, make sure to specify either an active or passive health check, or both, to avoid requests served to unhealthy backend.
+
+In presence of multiple upstreams, make sure to specify either an active or passive health check, or both, to avoid requests served to unhealthy backend.
```

Comment:
```suggestion
In the presence of multiple upstreams, make sure to specify either an active or passive health check, or both, to avoid requests being served to an unhealthy backend.
```